kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateExamPreview.vue

```diff
@@ -105,7 +105,7 @@
         :selectedExercises="selectedExercises"
       />
 
-      <BottomAppBar>
+      <BottomAppBar style="z-index: 1062;">
```

Comment:
where does this number magic come from?

This is fine for now, but generally, our z-indexes should map to Material elevations - https://material.io/design/environment/elevation.html#default-elevations